joblib/test/test_memory.py

```diff
@@ -225,6 +225,7 @@ def name_collision(x):
     with warnings.catch_warnings(record=True) as w:
         # Cause all warnings to always be triggered.
         warnings.simplefilter("always")
+        warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
Could you please open an issue for the `inspect.getargspec` deprecated calls in Python 3.5 and mention it here as an inline comment so that we can remove this line once fixed?